lambdas/docker-compose.yml

```diff
@@ -47,8 +47,8 @@ services:
     env_file:
       - ../.env.localstack
 
-  resources-import-producer:
-    image: resources-import-producer
+  resources-scheduled-importer:
+    image: resources-scheduled-importer
```

Comment:
Does not affects the fact that the image name is different from the folder that contains the source, and also from the lambda name in the Github actions? :thinking: